jdi-light-html-tests/src/test/java/io/github/epam/TestsInit.java

```diff
@@ -40,6 +35,6 @@ default void tearDown() {
     }
 
     default boolean isFireFox() {
-        return DRIVER.name.toLowerCase().equals("firefox");
+        return getJDISettings().DRIVER.name.toLowerCase().equals("firefox");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Using equalsIgnoreCase() is cleaner than using toUpperCase/toLowerCase().equals().](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5482899)